tinytroupe/extraction/artifact_exporter.py

```
@@ -0,0 +1,160 @@
+import os
+import json
+import pandas as pd
```

Ruff (0.8.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)